go/vt/vttablet/tabletmanager/vreplication/replica_connector.go

```diff
@@ -73,5 +73,5 @@ func (c *ReplicaConnector) Close() error {
 }
 
 func (c *ReplicaConnector) VStream(ctx context.Context, startPos string, filter *binlogdatapb.Filter, send func([]*binlogdatapb.VEvent) error) error {
-	return c.vstreamer.Stream(ctx, startPos, nil, filter, throttlerapp.ReplicaConnectorName, send)
+	return c.vstreamer.Stream(ctx, startPos, nil, filter, throttlerapp.ReplicaConnectorName, send, nil)
```

Comment:
Here again, based on how this is being used (only in PITR), we don't pass in the options.
Is it possible to replace usages of this connector with the tabletConnector?

Reply:
> Is it possible to replace usages of this connector with the tabletConnector?
I think so. Should be a separate PR though.

Reply:
We should even think about deleting all the binlog server code in the existing PITR implementation. No one is using it.